src/main/javassist/bytecode/Bytecode.java

```diff
@@ -1240,6 +1240,25 @@ public void addInvokedynamic(int bootstrap, String name, String desc) {
         growStack(Descriptor.dataSize(desc));   // assume ConstPool#REF_invokeStatic
     }
 
+    /**
+     * Appends DYNAMIC.
+     *
+     * @param bootstrap     an index into the <code>bootstrap_methods</code> array
+     *                      of the bootstrap method table.
+     * @param name          the method name.
+     * @param desc          the method descriptor.
+     * @see Descriptor#ofMethod(CtClass,CtClass[])
+     * @since 3.17
+     */
+    public void addDynamic(int bootstrap, String name, String desc) {
+        int nt = constPool.addNameAndTypeInfo(name, desc);
+        int dyn = constPool.addDynamicInfo(bootstrap, nt);
+        add(INVOKEDYNAMIC);
```

Comment:
Should this be DYNAMIC rather than INVOKEDYNAMIC?

Reply:
This looks like an alias of Bytecode#addInvokedynamic() in line 1225 (both bodies look the same).
This method addDynamic() has to be removed.

Reply:
I don't think there exits the bytecode instruction DYNAMIC.
So please remove this.